images/Dockerfile.test

```diff
@@ -0,0 +1,14 @@
+from golang:1.11-alpine3.8 as runner
+
+RUN apk add --update nodejs nodejs-npm
+RUN apk add git gcc libc-dev python make g++
```

Comment:
Let's get a comment in here for why each of these is added.

node for web3 and pusher.
git for the "go get" command below, I think.

gcc, libc-dev, g++, make, I think for the binaries built in web3. 

Not sure what python is for.

Reply:
git is also required by npm as it pulls down dependencies when installing node packages.

python is also for another web3 dependency. forgot which one.